universal-application-tool-0.0.1/test/controllers/applicant/ApplicantProgramsControllerTest.java

```diff
@@ -56,22 +55,21 @@ public void index_withNoPrograms_returnsEmptyResult() {
 
   @Test
   public void index_withPrograms_returnsOnlyRelevantPrograms() {
-    resourceCreator().insertProgram("one", LifecycleStage.ACTIVE);
-    resourceCreator().insertProgram("two", LifecycleStage.ACTIVE);
-    resourceCreator().insertProgram("three", LifecycleStage.DRAFT);
+    resourceCreator().insertActiveProgram("one");
+    resourceCreator().insertActiveProgram("two");
+    resourceCreator().insertDraftProgram("three");
 
     Result result =
         controller.index(fakeRequest().build(), currentApplicant.id).toCompletableFuture().join();
 
     assertThat(result.status()).isEqualTo(OK);
     assertThat(contentAsString(result)).contains("one");
     assertThat(contentAsString(result)).contains("two");
-    assertThat(contentAsString(result)).doesNotContain("three");
```

Comment:
Question: does the behavior change here?

Reply:
Hm, no, that should definitely still be there!  Re-adding.